app/models/student_identifier_type.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+class StudentIdentifierType < ActiveRecord::Base
+  default_scope { where(deleted_at: nil) }
```

Comment:
Hi @tarebyte, I wonder the reason why we need to ✂️ this. This line seems to be necessary for soft delete. It would be great if you can elaborate a little bit on this. Thanks!